libraries/AP_GPS/AP_GPS_SBF.h

```diff
@@ -106,9 +114,22 @@ class AP_GPS_SBF : public AP_GPS_Backend
          float VPL;
     };
 
+    struct PACKED msg4014 // ReceiverStatus (v2)
+    {
+         uint32_t TOW;
+         uint16_t WNc;
+         uint8_t CPULoad;
+         uint8_t ExtError;
+         uint32_t UpTime;
+         uint32_t RxState;
+         uint32_t RxError;
+         // remaining data is AGCData, which we don't have a use for, don't extract the data
+    };
+
     union PACKED msgbuffer {
         msg4007 msg4007u;
         msg4001 msg4001u;
+        msg4014 msg4014u;
```

Comment:
In my testing here it's always 48 bytes, leaves some headroom, but I couldn't find anywhere in the protocol sheet that limited the number of contained AGCData/padding. Assuming they pad out the AGCData to 4 byte we could fit in another 20 blocks of AGCData. I think that will be fine for the foreseeable future, but if you have an indication otherwise from the data sheets please let me know. :)